Cython/Compiler/ExprNodes.py

```diff
@@ -3085,22 +3103,39 @@ class CppIteratorNode(ExprNode):
 
     subexprs = ['sequence']
 
+    def __init__(self, pos, reversed=False, **kw):
+        super().__init__(pos, **kw)
```

Comment:
We're still supporting Python 2 unfortunately so `super` needs to be spelled out. You actually don't need this `__init__` though - the default behaviour for a node is to add all keyword arguments as attributes